library/idle.po

```diff
@@ -554,14 +553,18 @@ msgstr ""
 
 #: ../Doc/library/idle.rst:227
 msgid "Run... Customized"
-msgstr ""
+msgstr "Run... Customized"
 
 #: ../Doc/library/idle.rst:225
 msgid ""
 "Same as :ref:`Run Module <run-module>`, but run the module with customized "
 "settings.  *Command Line Arguments* extend :data:`sys.argv` as if passed on "
 "a command line. The module can be run in the Shell without restarting."
 msgstr ""
+"Similaire à :ref:`Run Module <run-module>`, mais il lance le module avec des "
+"paramètres personnaliés.  *Command Line Arguments* étend :data:`sys.argv` "
+"comme s'il était passé sur une ligne de commande. Le module peut être lancé "
```

Comment:
Ici "passed on a command line" signifie plutôt "passé par" que "passé sur", ça mériterai peut être une tournure plus habituelle comme : « Les valeurs de *Command Line Arguments* sont ajoutées à :data:`sys.argv` comme si elles avaient été données depuis une ligne de commande. »

Reply:
Tu veux dire qu'on remplace "run the module...with customized settings" par " ajoute les valeurs de Command Line Arguments à :data:sys.argv comme si elles avaient été entrées depuis une ligne de commande" ? Ca me semble un peu éloigné...

Reply:
Non, sans toucher à `run the module with customized settings`, ta traduction `lance le module avec des paramètres personnaliés.` est bonne.

C'est plus loin, dans : `comme s'il était passé sur une ligne de commande`, `passé sur` me semble trop litéral, en francais je n'etend pas qu'on "passe sur la ligne de commande", je proposais d'utiliser le verbe "donner" à la place, ou un autre synonyme, mais aussi utiliser "depuis la ligne de commande" au lieu de "sur la ligne de commande".

Reply:
Je vais garder le verbe "lancer" même si ca fait répétition avec la phrase d'après.